### DISCUSSION TITLE
Ability to create  Review Items by Camera Group  instead of by each camera

### DISCUSSION THREAD

**Discussion by easez88** [blakeblackshear/frigate#15576] [Ideas]
**Summary:**
Throughout Frigate you can view cameras together. For example, in the motion view  you can see multiple cameras and scrub through the footage at the same time. The camera group allows you to filter to specific cameras at any time. Reviews should take advantage of that.

**Problem:**  
Currently multiple reviews are triggered for the same capture if  cameras overlap each other. 

**Proposed Solution:**
It would be cool if the review section would follow camera groups and create 1 review containing multiple cameras. 

**Use Case:**
Say I have 2 cameras looking at the front of my house. A doorbell and an exterior camera. Currently, if a car or person passes by I get two reviews. 1 for the doorbell and 1 for the exterior camera. In reality, both cameras captured the same thing. 

With the proposed solution only 1 review could be created and when you click on it  you would see  both reviews, similar to motion. 

**Bonus:**
Object lifecycle  could also be enhanced to track when an object  entered the exterior camera and when it entered the doorbell camera. 

Users could also sync zones, by ensuring both cameras in the group have matching zones from their perspective.  So the Review would track when an object entered a zone, regardless of which camera it entered. 

Events would continue tracking by camera independently.

**Comment by hawkeye217** [comment]
There is already a feature request for a similar idea here: https://github.com/blakeblackshear/frigate/issues/14054

While not exactly your request, it relates in some ways.

**Reply by easez88** [reply]
I did take a look at that one and it is certainly close. The big differentiator being that even in a custom detection group, you would still get two reviews for different cameras. Where in this enhancement request you'd only get 1 in the  custom group. If both were added, it would be the customization options would be endless.
